### ISSUE TITLE
in re different output when recoil is set to true

### ISSUE THREAD

**Issue by latessa** [JETSCAPE/JETSCAPE#129]
I noticed that when recoil is set to true, running JETSCAPE with the same user settings produces different output.  Included below are the jetscape_user.xml file and three instances of different .hepmc output that resulted from this user xml file.  GitHub won't let me attach a .hepmc file, so I changed the file extensions to .txt

```
<?xml version="1.0"?>

<jetscape>
  
  <nEvents> 10 </nEvents>
  
  <JetScapeWriterHepMC> on </JetScapeWriterHepMC>
  <JetScapeWriterAscii> on </JetScapeWriterAscii>

  <Random>
    <seed>1</seed>
  </Random>
  
  <!-- Inital State Module  -->
  <IS>
    <Tren
[test_out_1.txt](https://github.com/JETSCAPE/JETSCAPE/files/9814785/test_out_1.txt)
[test_out_2.txt](https://github.com/JETSCAPE/JETSCAPE/files/9814786/test_out_2.txt)
[test_out_3.txt](https://github.com/JETSCAPE/JETSCAPE/files/9814787/test_out_3.txt)
to> </Trento>
  </IS>
  
  <!-- Hard Process -->
  <Hard>
    <PythiaGun>
      <pTHatMin>20</pTHatMin>
      <pTHatMax>30</pTHatMax>
      <eCM>5020</eCM>
    </PythiaGun>
  </Hard>
  
  <!--Preequilibrium Dynamics Module -->
  <Preequilibrium>
    <NullPreDynamics> </NullPreDynamics>
  </Preequilibrium>
  
  <!-- Hydro  Module  -->
  <Hydro>
    <Brick bjorken_expansion_on="false" start_time="0.6"> </Brick>
  </Hydro>
  
  <!--Eloss Modules -->
  <Eloss>
    <Matter>
      <in_vac> 0 </in_vac>
      <Q0> 2.0 </Q0>
      <recoil_on> 1 </recoil_on>
      <broadening_on> 0 </broadening_on>
      <qhat0> -2.0 </qhat0>
    </Matter>
    <LBT>
      <Q0> 2.0 </Q0>
    </LBT>
  </Eloss>
  
  <!-- Jet Hadronization Module -->
  <JetHadronization>
    <name>colorless</name>
  </JetHadronization>
  
</jetscape>
```

**Comment by latessa** [comment]
[test_out_1.txt](https://github.com/JETSCAPE/JETSCAPE/files/9814795/test_out_1.txt)
[test_out_2.txt](https://github.com/JETSCAPE/JETSCAPE/files/9814796/test_out_2.txt)
[test_out_3.txt](https://github.com/JETSCAPE/JETSCAPE/files/9814797/test_out_3.txt)

**Comment by Hendrik1704** [comment]
I think this issue is solved now. With JS 3.6 there is a fixed seed in the LBT module, such that the results should be reproducible now. Maybe you can redo this with the new version.

**Comment by latessa** [comment]
Great. I will rerun these tests and close the issue if the results are consistent. Thanks.

**Comment by latessa** [comment]
The output still differs with when recoil is turned on.

![output_differs](https://github.com/JETSCAPE/JETSCAPE/assets/11351196/d151d165-de03-47f7-b84f-aa12a81eca56)

**Comment by latessa** [comment]
PR #183 fixes this issue and is merged into the JETSCAPE 3.6.1 release candidate, so I'm marking this issue as closed.
